tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py{27, 36}
+
+[testenv]
+passenv = test_flags
+commands = python setup.py nosetests --xunit-file=nosetests_{envname}.xml {env:test_flags}
+
+deps =
+    coverage
+    mock
+    nose
+    nosexcover
+    future
```

Comment:
`future` is part of `install_requires` in `setup.py`, so no need for it here.  Removed.

Reply:
Was referring to `ipython`. Cant find the commit.